qa-pipelines/tasks/lib/cf-deploy-upgrade-common.sh

```diff
@@ -21,7 +21,7 @@ if [ -n "${CAP_INSTALL_VERSION:-}" ]; then
     export CAP_DIRECTORY=cap-install-version
     unzip ${CAP_DIRECTORY}.zip -d ${CAP_DIRECTORY}/
 else
-    unzip ${CAP_DIRECTORY}/scf-*.zip -d ${CAP_DIRECTORY}/
+    unzip ${CAP_DIRECTORY}/*scf-*.zip -d ${CAP_DIRECTORY}/
```

Comment:
why `*`?

Reply:
@prabalsharma because the name of this for pr builds will be `pr-xxxx-scf-...`

Reply:
where `xxxx` is the number of the PR (and also the parameter passed to the set-pipeline command)